Dockerfile

```diff
@@ -67,7 +67,7 @@ RUN apt-get update -qq && apt-get --no-install-recommends -yqq install \
     && rm -rf /data/cmake-${CMAKE_VERSION}.tar.gz \
     && echo "\e[32mbuilding: Boost\e[39m" \
     && set -ex \
-    && curl -s -L -o  boost_${BOOST_VERSION}.tar.bz2 https://dl.bintray.com/boostorg/release/${BOOST_VERSION_DOT}/source/boost_${BOOST_VERSION}.tar.bz2 > /dev/null \
+    && curl -s -L -o  boost_${BOOST_VERSION}.tar.bz2 https://boostorg.jfrog.io/artifactory/main/release/${BOOST_VERSION_DOT}/source/boost_${BOOST_VERSION}.tar.bz2 > /dev/null \
```

Comment:
bintray is being sunset, and jfrog is its replacement. See this MR for further context: https://github.com/monero-ecosystem/monero-javascript/pull/61